src/screens/OrganizationEvents/CustomRecurrenceModal.tsx

```
@@ -0,0 +1,596 @@
+        break;
+      case Frequency.MONTHLY:
+        updatedRule.byMonthDay = [currentMonthDay];
+        break;
+      case Frequency.YEARLY:
+        updatedRule.byMonth = [currentMonth];
+        updatedRule.byMonthDay = [currentMonthDay];
+        updatedRule.count = 5;
+        updatedRule.never = false;
+        updatedRule.endDate = undefined;
+        break;
+      case Frequency.DAILY:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)